packages/itwin/measure-tools/src/measurements/AreaMeasurement.ts

```diff
@@ -120,6 +121,8 @@ export class AreaMeasurement extends Measurement {
   constructor(props?: AreaMeasurementProps) {
     super();
 
+    if (props?.drawingMetaData)
```

Comment:
I think the `if (props) this.readFromJSON(props)` line should take care of that in the base Measurement class?